### DISCUSSION TITLE
Outdated Chia client on Spacepool message

### DISCUSSION THREAD

**Discussion by shnoah641** [Chia-Network/chia-blockchain#15089] [Support]
### What happened?

My client for Chia version 1.7.1 keeps telling me pool side on space pool that the version my chia client. it give me this error : 
This account is not receiving some or all of its rewards due to farming on at least one outdated Chia client
![Screenshot (2)](https://user-images.githubusercontent.com/72530034/233253897-07faccb9-3881-4d12-aa8b-7d89e67e701f.png)

### Version

1.7.1

### What platform are you using?

Windows

### What ui mode are you using?

GUI

**Comment by dutchmulti** [comment]
Having the same issue after update to 1.8 
![image](https://github.com/Chia-Network/chia-blockchain/assets/34837574/ad21bdd8-61ba-4fb3-b65d-fd9084bdacab)

**Reply by shnoah641** [reply]
It was due to me not being fully synced with the chia client that is why it read that but now it is working again.

**Comment by istorage1011** [comment]
Have the same issue after updating to 1.8.1. Node is Fully synced so i am not sure what the problem is.

**Reply by Olegik-ah** [reply]
I have same issue too. Chia 1.8.1 node fully synced

**Reply by shnoah641** [reply]
I had to do a full uninstall and then re-install of the software in order for it to correct itself sync wise. But use iobit uninstaller instead of the program manager.

**Reply by shnoah641** [reply]
Then run the program as administrator
